sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/FeedResponseDiagnostics.java

```diff
@@ -25,11 +26,13 @@ public class FeedResponseDiagnostics {
     private final static String EQUALS = "=";
     private final static String QUERY_PLAN = "QueryPlan";
     private final static String SPACE = " ";
+    private final static String FEED_RESPONSE_CREATION_LATENCY = "FeedResponseCreationLatency";
```

Comment:
Can we simplify this to 
```suggestion
    private final static String FEED_RESPONSE_LATENCY = "FeedResponseLatency";
```